llvm/test/CodeGen/AIE/aie2/schedule/resource/vst_pack.mir

```diff
@@ -0,0 +1,33 @@
+# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py
```

Comment:
Nit: Test file should probably be named w_rs_port.mir

Reply:
Nice, we have per resource tests!